src/components/pages/paymentPage/PaymentPage.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ChangeEvent, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import Layout from '../../layout/Layout';
+import { Button } from 'reactstrap';
+import './paymentPage.scss';
+import Input from '../../common/Input';
+
+const PaymentPage = () => {
+  const { t } = useTranslation();
+  const [termsAccepted, setTermsAccepted] = useState<boolean>(false);
+
+  const handleAcceptTermsChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const target = event.target;
+    setTermsAccepted(target.checked);
+  };
+
+  const handlePay = () => {
+    alert('payment');
+  };
+
+  return (
+    <Layout>
+      <div className="vene-payment-page">
+        <div className="vene-payment-page__content-container">
+          <div className="vene-payment-page__content">
+            <h2>{t('page.payment.title')}</h2>
+            <p>{t('page.payment.terms_info')}</p>
+            <p className="vene-payment-page__contact-info">
+              {t('page.payment.questions')}&nbsp;
+              <a href="mailto:venepaikat@hel.fi" className="vene-payment-page__link">
+                venepaikat@hel.fi
+              </a>
+            </p>
+          </div>
+        </div>
+        <div className="vene-payment-page__content-container">
```

Comment:
The upper padding of this component is a little too much when compared to the design in Abstract

Reply:
done